src/ILVerification/tests/ILVerification.Tests.csproj

```diff
@@ -3,16 +3,21 @@
 
   <PropertyGroup>
     <OutputType>Library</OutputType>
+    <TargetFramework>netstandard1.5</TargetFramework>
```

Comment:
Does this need anything from 1.5 ? I would keep it at netstandard1.3.

Reply:
I was getting errors when doing `msbuild /t:Restore` that `Microsoft.TestPlatform.TestHost 15.3.0-preview-20170427-09` requires netstandard 1.5. I'll try again, to confirm.

Reply:
Tried again and confirmed. That said, I'm not exactly sure how that package is getting pulled into the graph...

```
F:\repos\corert>msbuild /v:m /m  "F:\repos\corert\src\ILVerification\tests\ILVerification.Tests.csproj" /t:Restore
Microsoft (R) Build Engine version 15.6.82.30579 for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

  Restoring packages for F:\repos\corert\src\ILVerification\tests\ILVerification.Tests.csproj...
  Restore completed in 56.61 ms for F:\repos\corert\src\ILVerification\src\ILVerification.csproj.
F:\repos\corert\src\ILVerification\tests\ILVerification.Tests.csproj : error NU1202: Package Microsoft.TestPlatform.TestHost 15.3.0-preview-20170427-09 is not compatible with netstandard1.3 (.NETStandard,Version=v1.3). Package Microsoft.TestPlatform.TestHost 15.3.0-preview-20170427-09 supports:
F:\repos\corert\src\ILVerification\tests\ILVerification.Tests.csproj : error NU1202:   - net45 (.NETFramework,Version=v4.5)
F:\repos\corert\src\ILVerification\tests\ILVerification.Tests.csproj : error NU1202:   - netstandard1.5 (.NETStandard,Version=v1.5)
  Generating MSBuild file F:\repos\corert\bin/obj/Windows_NT.x64.Debug/ILVerification.Tests\ILVerification.Tests.csproj.nuget.g.props.
  Restore failed in 529.32 ms for F:\repos\corert\src\ILVerification\tests\ILVerification.Tests.csproj.
```

Reply:
ok